src/pyggdrasil/analyze/_metrics.py

```diff
@@ -15,7 +15,7 @@ def get(metric: str) -> Callable[[TreeNode, TreeNode], float]:
         return Metrics._METRICS[metric]
 
     _METRICS: Dict[str, Callable[[TreeNode, TreeNode], float]] = {  # type: ignore
-        "AD": AncestorDescendantSimilarity().calculate,
+        "AD": AncestorDescendantSimilarity_lq().calculate,
```

Comment:
Making Laura's the default -  refactoring the `*_lq` away once we know if `scyphylo` is indeed broken